Bio/bgzf.py

```diff
@@ -628,63 +628,65 @@ def seek(self, virtual_offset):
         #       self._within_block_offset)
         return virtual_offset
 
-    def read(self, size=-1):
+    def read(self, _size=-1):
```

Comment:
Why are you changing the argument name? That's a backward incompatible API change.